### PR TITLE
Brotli seems to be required for this version else libbrotlienc is missing

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -23,6 +23,7 @@ dependencies = [
     ('GLib', '2.72.1'),
     ('GnuTLS', '3.7.8'),
     ('libidn2', '2.3.2'),
+    ('Brotli', '1.0.9'),
```
Comment (branfosj) [inline easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.3.0.eb:26]: This surprised me. Can you share the error / build log where it fails?

I've checked `ldd` output on the `glib-networking` `.so` files and I cannot see signs of Brotli.

Comment (sassy-crick) [reply easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.3.0.eb:26]: Apologies for the delay.
It surprised me as well and it was only when I inspected the meson-log.txt I found that:
```
pkg-config error with 'gnutls': Could not generate cargs for gnutls:
Package libbrotlienc was not found in the pkg-config search path.
Perhaps you should add the directory containing `libbrotlienc.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libbrotlienc', required by 'gnutls', not found
Package 'libbrotlidec', required by 'gnutls', not found
```
The full log files, i.e. from EasyBuild and Ninja, are [here](https://gist.github.com/sassy-crick/8c93a3b032bc93f501c99e8e365476f4). 

Please let me know if you got a better solution for that and thanks for your help.

Comment (branfosj) [reply easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.3.0.eb:26]: So the issue is it is not finding your `GnuTLS` install, but I am confused as to why it is expecting to find `libbrotlienc`.

```
$ module load GnuTLS/3.7.8-GCCcore-11.3.0 pkgconf/1.8.0-GCCcore-11.3.0
$ pkg-config --modversion gnutls
3.7.8
$ pkg-config --cflags gnutls
-I/rds/projects/2017/branfosj-rse/easybuild/EL8-ice/software/p11-kit/0.24.1-GCCcore-11.3.0/include/p11-kit-1 -I/usr/local/include
$ echo $?
0
```

Can you try those commands and also run `ldd` on the GnuTLS .so files.

**Comment by Flamefire** [comment]
From current Slack discussion:
@sassy-crick has a GnuTLS pkgconfig with:
```
Name: GnuTLS
Description: Transport Security Layer implementation for the GNU system
URL: https://www.gnutls.org/
Version: 3.7.8
Libs: -L${libdir} -lgnutls
Libs.private:        -lgmp -lunistring -latomic 
Requires.private: nettle, hogweed, libtasn1, p11-kit-1, zlib, libbrotlienc, libbrotlidec, libzstd
Cflags: -I${includedir}
```

On e.g. my system the `libbrotli*` libs are not there, so the Brotli dependency would be for GnuTLS but first it would be required to determine why on his system that dependency is added in the first place.

**Pull request closed without merging**